src/modules/users/page_usersetup.ui

```diff
@@ -460,6 +460,13 @@
      </property>
     </spacer>
    </item>
+   <item>
+    <widget class="QCheckBox" name="checkBoxValidatePassword">
+     <property name="text">
+      <string>Require strong passwords.</string>
```

Comment:
I expect -- without having compiled this to see what it looks like -- that this is going to add clutter to an already cluttered form. Can we get a designer to take a look at this? I suspect we might end up with a dots-menu about passwords.